src/Executables/Game/ShopsManager.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.Extensions.Logging;
+using QuantumCore.API;
+using QuantumCore.API.Core.Models;
+using QuantumCore.Game.Persistence;
+
+namespace QuantumCore.Game;
+
+public interface IShopsManager
+{
+    public Task<ShopItem[]> GetShopItems(uint npcVnum);
```

Comment:
can't you use dynamical arrays instead of static arrays?

Reply:
What is "dynamic" and what is "static" from ur point of view?